hvplot/converter.py

```diff
@@ -819,6 +826,17 @@ def _process_data(self, kind, data, x, y, by, groupby, row, col,
             data, x, y, by_new, groupby_new = process_xarray(
                 data, x, y, by, groupby, use_dask, persist, gridded,
                 label, value_label, other_dims, kind=kind)
+            if robust:
```

Comment:
Shouldn't this block only run for RGB elements?

Reply:
In hvPlot if I now set `robust=True` to display an Image element, I think that's going to combine the rescaling you're doing in this block, together with `clim_percentile=True`. Is that expected?

Reply:
Ah, you're right about the scaling part.